calyx/src/passes/cell_share.rs

```diff
@@ -215,6 +234,9 @@ impl CellShare {
             })
             .collect();
 
+        let print_par_timing =
+            given_opts.iter().any(|arg| *arg == "print_par_timing");
```

Comment:
Why do you need to explicitly parse this? The `get_opts` function should be able to parse this for you. For example: https://github.com/cucapra/calyx/blob/master/calyx/src/passes/top_down_static_timing/tdst.rs#L751

Reply:
Ah, I'm just realizing, if I use `get_opts()` then it will give me a warning for flags like "-x cell-share:bounds=2,4,-1", since `get_opts()`.